### PR TITLE
Modify file_integration_using_smb module.md file

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Modify file_integration_using_smb module.md file

### CODE REVIEW

File: docs/content/src/learn/tutorials/file-based-integrations/file-integration-using-smb/src/file_integration_using_smb/Module.md [hunk 1]
```diff
@@ -1,109 +1,60 @@
...
-        wso2/file_integration_using_smb:0.1.0
-
-Creating balos
-        target/balo/file_integration_using_smb-2019r3-java8-0.1.0.balo
-```
-
-This will create the Ballerina executables inside the `/target` directory.
-
-Then run the jar file created in the above step.
+This creates the executables. Now run the `file_integration_using_smb.jar` file created in the above step.Path to the ballerina.conf file can be provided using the --b7a.config.file option.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/file-based-integrations/file-integration-using-smb/src/file_integration_using_smb/Module.md]:
```suggestion
This creates the executables. Now run the `file_integration_using_smb.jar` file created in the above step. Path to the ballerina.conf file can be provided using the --b7a.config.file option.
```

File: docs/content/src/learn/tutorials/file-based-integrations/file-integration-using-smb/src/file_integration_using_smb/Module.md [hunk 2]
```diff
@@ -1,109 +1,60 @@
...
-Create the service to listen to the Samba server using the above listener. When files are added or deleted on the server,
-this service will be invoked, and the files will be processed.
+### Before you begin
 
-Then implement the Samba Client, which will connect to the Samba server and get the details of new files to process.
+### 1. Add project configurations file
 
-Declare a map to store the details of processed files.
+Add the project configuration file by creating a `ballerina.conf` file under the root path of the project structure. 
+This file should have following configurations. Add the FTP server configurations to the file.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/file-based-integrations/file-integration-using-smb/src/file_integration_using_smb/Module.md]:
```suggestion
This file should have following configurations. Add the SMB server configurations to the file.
```

File: docs/content/src/learn/tutorials/file-based-integrations/file-integration-using-smb/src/file_integration_using_smb/Module.md (hunk 1 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/file-based-integrations/file-integration-using-smb/src/file_integration_using_smb/Module.md]: Fixed

File: docs/content/src/learn/tutorials/file-based-integrations/file-integration-using-smb/src/file_integration_using_smb/Module.md (hunk 2 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/file-based-integrations/file-integration-using-smb/src/file_integration_using_smb/Module.md]: Fixed

**Pull request merged**